apps/src/gamelab/AnimationPicker/AnimationPreview.jsx

```diff
@@ -133,7 +138,11 @@ const AnimationPreview = React.createClass({
     );
   }
 });
-export default AnimationPreview;
+module.exports = connect(function propsFromStore(state) {
+  return {
+    allAnimationsSingleFrame: state.pageConstants.allAnimationsSingleFrame
+  };
+})(AnimationPreview);
```

Comment:
I'm moderately attached to the idea that this is a display-only component, not connected to global state in any way.  I wonder if, instead of making the `AnimationPreview` care about the `allAnimationsSingleFrame` property, it can only care about whether it has been told to animate.

This might mean we now have a sort of 'trinary' property for when to animate - there's the default "play on hover" behavior, the "always play" behavior denoted by `alwaysPlay: true`, and now a "never play" behavior.  What if we replaced `alwaysPlay` with a `playBehavior` enum?  That would eliminate the funny edge case right now where we could set both `alwaysPlay` and `allAnimationsSingleFrame`.

Thoughts?